projects/cds/src/profiletag/services/profile-tag.injector.ts

```diff
@@ -16,19 +16,22 @@ import {
   take,
   tap,
 } from 'rxjs/operators';
-import { CdsConfig } from '../../config/cds-config';
+import { CdsConfig } from '../../config';
 import {
   ProfileTagEvent,
+  ProfileTagEventNames,
   ProfileTagJsConfig,
   ProfileTagWindowObject,
-} from '../model/profile-tag.model';
+} from '../model';
```

Comment:
Please specify the full path of the file, i.e `... from '../model/profile-tag.model';`

Reply:
Will do. I noticed that you added the index.ts files to all of these folders, and I thought the point was so that we don't have to specify the full path, and just use the index.ts, but I will switch it ;)